spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/AbstractSpannerIntegrationTest.java

```diff
@@ -113,28 +113,31 @@ public void setup() {
 		}
 	}
 
+	@Test
```

Comment:
Now it's really vital to make `setup()` `@BeforeClass`.

Reply:
Oooh i understand now. Yes will do.

Reply:
@meltsufin 

I switched to using `@BeforeClass and @AfterClass`, however the requirement is that these both be static methods. I went and changed all the things they need to also be static, but it doesn't seem like ApplicationContext is set by Spring before `@BeforeClass` can be called. I tried both using `@Autowire` application context and also by implementing ApplicationContextAware, but neither work.

I found somebody on SO had the same problem and he noted neither solution worked for him either: 
https://stackoverflow.com/questions/29496643/how-to-access-spring-applicationcontext-in-junit-beforeclass-static-method

Instead I found an alternative of using some static booleans that keep track of if the DB has been initialized and the regular non-static `@Before` set up method just checks that static boolean :
https://stackoverflow.com/questions/17505952/how-to-populate-database-only-once-before-test-methods-in-spring-test

Reply:
The static boolean works I guess.

Reply:
@meltsufin OK, changes are up. PTAL